README.md

```diff
@@ -42,6 +42,7 @@ Thanks goes to these wonderful people
 ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+
```

Comment:
That one is odd, it should go before `<!-- ALL-CONTRIBUTORS-LIST:END -->`

Reply:
Updated!